authlib/oidc/core/claims.py

```diff
@@ -56,7 +56,7 @@ def validate_auth_time(self):
         if self.params.get('max_age') and not auth_time:
             raise MissingClaimError('auth_time')
 
-        if auth_time and not isinstance(auth_time, int):
+        if auth_time and not (isinstance(auth_time, int) or isinstance(auth_time, float)):
```

Comment:
`isinstance(auth_time, (int, float))`

Reply:
sorry, I have few to no experience with python. applied this simplification.